targets/REV3-LVSS/main.cpp

```diff
@@ -48,13 +48,13 @@ int main() {
     log::LOGGER.setUART(&uart);
     log::LOGGER.setLogLevel(log::Logger::LogLevel::INFO);
 
-    IO::GPIO& lvssEn0 = IO::getGPIO<IO::Pin::PA_10>(IO::GPIO::Direction::OUTPUT);
+    IO::GPIO& lvssEn0 = IO::getGPIO<IO::Pin::PA_10>(IO::GPIO::Direction::INPUT);
```

Comment:
Rename these variables to make it clear that they're for enabling the power switches. Also, shouldn't these be set to outputs?